generator-mediator-js/app/templates/mediatorTemplate/lib/_index.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+'use strict'
+
+const express = require('express')
+const medUtils = require('openhim-mediator-utils')
+
+const utils = require('./utils')
+const OpenHIM = require('./openhim')
+
+// Config
+var config = {} // this will vary depending on whats set in openhim-core
+const apiConf = require('../config/config')
+const mediatorConfig = require('../config/mediator')
+
+// socket config - large documents can cause machine to max files open
+const https = require('https')
+const http = require('http')
+
+https.globalAgent.maxSockets = 5
+http.globalAgent.maxSockets = 5
```

Comment:
The above 4 lines can be removed, this was needed because of a RapidPro server issue.